### PR TITLE
chore(dev workflow, readme): consistency commit

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
-# Documentation Search
+# docsearch.js
 
-Add a search autocomplete to your documentation.
+Add a search autocomplete to any documentation.
+
+Currently on-demand, send an email to [docsearch.js@algolia.com](mailto:docsearch.js@algolia.com)
```
Comment (pixelastic) [inline README.md]: Maybe just docsearch@, not docsearch.js@. The js part is just the front-end, but could not work without the backend crawler.

File: scripts/build-css [hunk 2]
```diff
@@ -3,7 +3,7 @@
 set -e # exit when error
 
 VERSION=$(json version < package.json)
-NAME='docsearch'
+NAME='docsearch.js'
 LICENSE="/*! ${NAME} ${VERSION:-UNRELEASED} | © Algolia | github.com/algolia/docsearch */"
```
Comment (pixelastic) [inline scripts/build-css]: We need to update the url in the license as well

File: scripts/build-js [hunk 3]
```diff
@@ -3,7 +3,7 @@
 set -e # exit when error
 
 VERSION=$(json version < package.json)
-NAME='docsearch'
+NAME='docsearch.js'
 LICENSE="/*! ${NAME} ${VERSION:-UNRELEASED} | © Algolia | github.com/algolia/docsearch */"
```
Comment (pixelastic) [inline scripts/build-js]: Same ^

File: README.md [hunk 4]
```diff
@@ -35,16 +38,36 @@ The JavaScript library is a wrapper on top of our
 [autocomplete.js](https://github.com/algolia/autocomplete.js) library, along
 with default CSS styling of the dropdown.
 
-The indexing of the data itself is currently done by an internal tool (we will
-release it later), that runs every hour.
+The indexing of the website data itself is currently done by an internal tool (to be released).
+
+Indexing of docsearch.js linked websites takes places every hour.
```
Comment (pixelastic) [inline README.md]: Let's just say once a day for now (this is how the Scala one is configured)

**Comment by vvo** [comment]
@pixelastic fixed your comments

File: README.md [hunk 5]
```diff
@@ -35,16 +38,36 @@ The JavaScript library is a wrapper on top of our
...
-release it later), that runs every hour.
+The indexing of the website data itself is currently done by an internal tool (to be released).
+
+Indexing of docsearch.js linked websites takes places every day.
+
+# Development workflow
+
+## Local example
+
+We use this as our primilary way to develop the docsearch.js library.
```
Comment (pixelastic) [inline README.md]: s/primilary/primary/ :)

**Pull request merged**